src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -136,7 +136,7 @@ walletApplication
     -> m Application
 walletApplication serv = do
     wsConn <- getWalletWebSockets
-    serv >>= return . upgradeApplicationWS wsConn . serve walletApi
+    upgradeApplicationWS wsConn . serve walletApi <$> serv
```

Comment:
I kinda don't like such use of <&> because it breaks value flow – we go left-to-right, then suddenly right-to-left. If somebody else likes <&> here then I'll remember that my intuitions about readability don't extend to other people, but for now I'm inclined to leave it as-is.